airflow/www/templates/airflow/ti_log.html

```diff
@@ -121,10 +121,18 @@ <h4>{{ title }}</h4>
           if (res.message) {
             // Auto scroll window to the end if current window location is near the end.
             if(auto_tailing && checkAutoTailingCondition()) {
-              var should_scroll = true
+              var should_scroll = true;
             }
             // The message may contain HTML, so either have to escape it or write it as text.
-            document.getElementById(`try-${try_number}`).textContent += res.message + "\n";
+            var escaped_message = $("<div/>").text(res.message).html();
+
+            // Detect urls
+            var url_regex = /(https?:\/\/[^\s<>"]+)/g;
```

Comment:
Will this cover all cases?

How about the following from http://www.noah.org/wiki/RegEx_Python?

```
http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+
```

Reply:
This regex doesn't work when there's a anchor tag (#). I switched to another more restrict regex - https://www.regexpal.com/?fam=111806